### PR TITLE
https://wso2.org/jira/browse/BPS-429

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+		System.setProperty("javax.net.ssl.trustStore", trustStore);
+		System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
+
+		System.out.println(CarbonUtils.getCarbonHome());
+		ConfigurationContext ctx =
+				ConfigurationContextFactory.createConfigurationContextFromFileSystem(
+						CarbonUtils.getCarbonHome() +
+						File.separator + "repository" + File.separator + "deployment" +
+                         File.separator + "client", null);
```
Comment (nandika) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Fix code formatting

File: modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+		String trustStore = CarbonUtils.getCarbonHome() + File.separator + "repository" + File.separator + "resources" +
+		                    File.separator + "security" + File.separator + "wso2carbon.jks";
+		String clientKey = trustStore;
+		OMElement result;
+
+		System.setProperty("javax.net.ssl.trustStore", trustStore);
+		System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
+
+		System.out.println(CarbonUtils.getCarbonHome());
```
Comment (chrishantha) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Avoid System.out.println(). Use the log

Comment (isurusuranga) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Fixed in https://github.com/wso2/product-bps/pull/100

File: modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java (hunk 1 above)
Comment (isurusuranga) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Fixed in https://github.com/wso2/product-bps/pull/100

**Pull request merged**